features/support/sidekiq.rb

```diff
@@ -4,7 +4,7 @@
 require 'sidekiq/batch'
 
 # Turn off Sidekiq logging which pollutes the CI logs
-Sidekiq::Logging.logger = nil
+Sidekiq.logger = Logger.new("/dev/null")
```

Comment:
I cringe when I see platform specific things although I don't think we run on anything but linux. Also according to docs, better use

```suggestion
Sidekiq.logger = Sidekiq::Logger.new(IO::NULL, level: Logger::FATAL)
```

https://github.com/sidekiq/sidekiq/wiki/Logging

Reply:
actually, why not
```suggestion
Sidekiq.logger = Sidekiq::Logger.new(nil, level: Logger::FATAL)
```